monitor/dbt_project/macros/get_new_test_alerts.sql

```diff
@@ -1,53 +1,37 @@
 {% macro get_new_test_alerts(days_back, results_sample_limit = 5) %}
-    -- depends_on: {{ ref('alerts_tests') }}
+    -- depends_on: {{ ref('alerts') }}
     {% set select_new_alerts_query %}
-        WITH alerts AS (
-            SELECT * FROM {{ ref('alerts_tests') }}
-            WHERE alert_sent = FALSE and detected_at >= {{ get_alerts_time_limit(days_back) }}
+        with new_alerts as (
```

Comment:
cosmetics - let's rename the cte to new_tests?

Reply:
In the context of `get_new_test_alert()`, the name makes sense, don't you think?